apps/tanstack-start-app/README.md

```diff
@@ -0,0 +1,72 @@
+# Welcome to TanStack.com!
```

Comment:
I don't really know what's going on with this README but it doesn't seem relevant